acoustid/future/fpindex/client.py

```
@@ -0,0 +1,396 @@
+    async def close(self) -> None:
+        """Close the HTTP session"""
+        await self.session.close()
+
+    def _url(self, path: str) -> str:
+        """Build a URL for the given path"""
+        return urljoin(self.base_url, path)
+
+    T = TypeVar("T", bound=msgspec.Struct)
+
+    @overload
+    async def _request(  # NOQA: E704
```

Ruff (0.8.2)

Unused `noqa` directive (unknown: `E704`)

Remove unused `noqa` directive

(RUF100)